_docs/cardano/2017-02-07-topology.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Topology
+permalink: /cardano/topology/
+group: cardano
+---
+<!-- Reviewed at dec0d911d6c4beb8e708ed4076f832ff871f6125 -->
+
+# Cardano SL Network Topology
+
+This is an overview of Cardano SL network topology.
+
+## Nodes Groups
+
+Cardano SL is a distributed network: it consists of nodes. But currently all these nodes are divided into
+3 groups:
+
+1. Core
+2. Relay
+3. Edge
+
+Let's describe each of these groups.
+
+### Core Node
+
+You can think of core nodes as of Cardano SL core.
+
+These nodes are the most important ones: if they are alive, the blockchain cannot be compromised. So core
+nodes are isolated: they are fully under the control of the IOHK and cannot be directly accessible from
```

Comment:
> they are fully under the control of the IOHK and cannot be directly accessible

These details should not be here, they relate to description of bootstrap era (or Byron release).

What we need to say here they're isolated and put in perimeter of relay nodes, and only relay nodes are allowed to communicate to core nodes